core/templates/services/assets-backend-api.service.ts

```diff
@@ -66,6 +67,8 @@ export class AssetsBackendApiService {
     }
     this.downloadUrlTemplate = (
       urlPrefix + '/<entity_type>/<entity_id>/assets/<asset_type>/<filename>');
+    this.profileImageUrlTemplate = (
+      urlPrefix + '/user/<username>/profile_image.png');
```

Comment:
NOTE: This is the URL that will be directly queried to get the user's profile image in GCS. (`url_prefix` is a GCS URL)